packages/ts-fdc3-oms/src/hooks/useInstrumentDetailsContext.tsx

```diff
@@ -84,27 +94,32 @@ const useInstrumentDetailsContext = (): UseInstrumentDetailsContext => {
                     setCtxSymbol(contextData)
                 })
                 .catch(console.error)
-        }
+        } else if (onChannel) {
```

Comment:
could you please refrain from using if-else statements, as it is generally discouraged. 
Stick to the established "If(condition) {return}" pattern